src/skmatter/metrics/__init__.py

```diff
@@ -39,7 +39,7 @@
   linear or kernel model.
 
 There are also two distance metrics compatible with the periodic boundary conditions
-available (now only rectangular box supported):
+available (currently only rectangular cells are supported):
```

Comment:
Maybe also here a note for the cell

```rst
.. note::
    currently only rectangular cells are supported
```

Another note. For me this is fine if we support only rectangular cell.  But, if you have capacities general cells would be nice in the future!